csharp/src/Ice/IEndpointFactory.cs

```diff
@@ -30,7 +30,8 @@ public interface IEndpointFactory
         /// <summary>Reads an endpoint from the given input stream.</summary>
         /// <param name="istr">The input stream to read from.</param>
         /// <param name="protocol">The protocol of the enclosing proxy.</param>
+        /// <param name="transport">The transport of the endpoint.</param>
```

Comment:
I think it would be more logical to list the parameters as transport, protocol - transport is more essential here.

Reply:
Fixed the order